console/src/main/resources/org/togglz/console/index.html

```diff
@@ -60,7 +60,7 @@
                   <ul class="params">
                   ${foreach feature.strategy.parameters param}
                     ${if param.hasValue}
-                      <li>${param.label}: ${param.value}</li>
+                      <li>${param.label}: ${param.value;sanitizeHtml}</li>
```

Comment:
@BeLi4L does this work? That looks wrong somehow

Reply:
Yet that's the official JMTE syntax:
- https://www.tinymediamanager.org/docs/jmte#renderer-information
- https://code.google.com/archive/p/jmte/wikis/BestOfAPI.wiki

It uses the renderer name (here `sanitizeHtml`) defined in `SanitizeHtmlRender.java`

Reply:
And yes it works, I tested on my machine, the custom renderer is called, and it prevents XSS on the main page, when displaying the script ;)

Reply:
Okay thanks :) the syntax looks a bit weird.
Thanks for your PR :)